independent-publisher-connectors/Doppler/apiProperties.json

```diff
@@ -0,0 +1,96 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Service Token",
+          "description": "The Service Token for this api",
+          "tooltip": "Provide your Service Token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#5e17eb",
```

Comment:
Please update the iconBrandColor as per the template here?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json